test/spec/functional/viewTransition.js

```diff
@@ -0,0 +1,59 @@
+var testSrcPath = '../../source',
+    spyOnBackbone = require('./backboneSpy');
+
+describe('A View that can transition in tracked views', function() {
+
+  var env, _, $, ParentView;
+
+  beforeEach(function(done) {
+    require('./clientEnv')().done(function(environment) {
+      env = environment;
```

Comment:
local vars using this is best practice with Jasmine (as far as I know).

this.env = environment;
this.$ = ...
...

Then you can access the vars in your it and describe blocks with this.env, etc.  This prevents any leaking state from crossing tests.